tests/unit/TodoList.spec.ts

```diff
@@ -1709,6 +1763,103 @@ describe('TodoList - Click Tests - BCOMPs', () => {
   })
 })
 
+describe('TodoList - Click Tests - NRs and Incorp Apps', () => {
+  const { assign } = window.location
+
+  beforeAll(() => {
+    // init store
+    sessionStorage.setItem('CREATE_URL', `${process.env.VUE_APP_PATH}/create/`)
```

Comment:
Not quite the correct URL but doesn't matter for the unit tests.
This will probably be more correct in the future when we clean up our URLs :)